lib/rubocop/cop/layout/empty_line_between_defs.rb

```diff
@@ -13,6 +13,10 @@ module Layout
       # `AllowAdjacentOneLineDefs` configures whether adjacent
       # one-line method definitions are considered an offense.
       #
+      # `ApplyToClassDefs` checks for empty lines around class definitions.
```

Comment:
I think those names and descriptions are misleading, as there is already a cop checking for empty lines around class bodies.

Reply:
I mean mostly in the way that when enabled those settings should also cover different definition types as in the issue that was reported - e.g. you'll want space between a module definition and a method definition and so on. The documentation needs to be expanded and more examples should be added.